x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/components/donut_chart.tsx

```diff
@@ -31,7 +31,7 @@ export const DonutChart = ({ height, width, data }: DonutChartProps) => {
         .ordinal()
         // @ts-ignore
         .domain(data)
-        .range(['#017D73', '#98A2B3', '#BD271E']);
+        .range(['#017D73', '#98A2B3', '#BD271E', '#9b6900']);
```

Comment:
pulled official EUI warning color from here: https://elastic.github.io/eui/#/guidelines/colors

```suggestion
        .range(['#017D73', '#98A2B3', '#BD271E', '#F5A700']);
```